pdf/config.json5

```diff
@@ -1067,13 +1067,19 @@ with open(sys.argv[1], 'w') as html: \n\
                     // Similarly, {disttarg} gets rewritten with '/home/dist', BUT
                     // it supports modular configs (those in child folders).
                     'COPY {dist}/qpdf_plugin/requirements.txt /home/dist/qpdf_plugin/requirements.txt',
-                    'RUN pip3 install -r /home/dist/qpdf_plugin/requirements.txt',
+                    'RUN pip3 install --trusted-host pypi.org --trusted-host files.pythonhosted.org  -r /home/dist/qpdf_plugin/requirements.txt',
 
                     'COPY {dist}/schizo_test/requirements.txt /home/dist/schizo_test/requirements.txt',
-                    'RUN pip3 install -r /home/dist/schizo_test/requirements.txt',
+                    'RUN pip3 install --trusted-host pypi.org --trusted-host files.pythonhosted.org  -r /home/dist/schizo_test/requirements.txt',
 
                     'COPY {dist}/ml_test/requirements.txt /home/dist/ml_test/requirements.txt',
-                    'RUN pip3 install -r /home/dist/ml_test/requirements.txt',
+                    'RUN apt-get install -y ca-certificates',
+                	'COPY BAESystems-Inc-Root-Base64.crt /usr/local/share/ca-certificates/BAESystems-Inc-Root-Base64.crt',
```

Comment:
These BAESystems-specific lines aren't a good idea to include in a public Dockerfile. One could use a local file that doesn't get checked in to git as a bypass, but I wonder if the real solution isn't to see if BAE IT has a more transparent fix? This seems to be caused by some network proxying behavior. If that's the case, then the proxy is configured incorrectly if it requires lines like these to be added.